ALiAS_FAQs.md

```diff
@@ -0,0 +1,47 @@
+                                                FAQs
+Q1. What is ASET AliAS?
+Ans. It’s not your regular Amity Club. ALiAS is a community, founded in 2010, which was developed with the aim to promote the use of Linux in the campus and get students familiar with the Open Source. We community members help students with their doubts pertaining tech, conduct workshops, webinars, have meet-ups (online and in-person) and most important get you connected.
+
+Q2. Why should I join ALiAS?
+Ans. ALiAS is by far, the largest open-source community in Amity University, we are
+open to developers not only inside the campus, but also from various places. We have got assistance
+from alumni and people from tech, near and far. We have got presence in communities like PyDelhi, ILUG-D, LinuxChix India, Mozilla Delhi, etc. and can help you get connected.
+
+Q3. How is the community of AliAS?
+And. We have got the most active developers in the campus. You will find people in nearly all domains,
+Web Development, Android Development, Artificial Intelligence, backend, frontend, industry people,
+etc. And best of all, we have got a great reach for people and further development.
+
+Q4. How should I join AliAS?
+And. Simple. Go and register yourself in http://asetalias.in/ . One of our developers will get you joined
+to the group.
+
+Q5. Having joined, where should I start?
+And. We request you to drop a message, stating your name, course duration and your skills, if any. Also
+let us know which field you want to work, and we will guide you thereafter. Have one plan at a time.
+
+Q6. Best OS?
+Ans. Are you a designer? - Windows; Are you a developer? - Linux(Any)
+Windows has great graphics. It spoon feeds you everything. Many design softwares, games, drivers and
+support is easily available in windows. Installing anything is a cup of cake for anyone. But if you want
+to delve into softwares deeper, please forget your games and ease. Being open source community, we
+recommend installing Linux as dual boot, and get into ease with it. The only difference you will see in
+different distros is the GUI. Some software packages and libraries are pre-installed in some distros,
+while some requires you to install. Install Ubuntu or Linux Mint if you are a beginner. Ubuntu has got
+best support for you. (Pro-tip: Give more space to Home directory as all your files are installed here).
+
+Q7. The language I should start with?
+Ans. Start with C++. Next learn JAVA, then Python. This will fulfill most of your software needs and
```

Comment:
```suggestion
Ans. Start with C++. Next learn JAVA, then Python. This will fulfill most of your software needs and
```
Broo, C++ is **NOT** a good first language.
It's too complex for beginners.
Start with a general purpose, easy to learn, currently in fashion language.
Then maybe suggest 2nd languages based on their domain of choice

Reply:
Will C work?

Reply:
Ahh...NO, C is worse than C++ 
Try something easy like python/ruby/javascript, some scripting language
C++ and Java are good for 2nd language when people have some idea what they're doing
C is the real s***. Ir's for later :rofl: